### ISSUE TITLE
Failed on macos

### ISSUE THREAD

**Issue by zakirovir6** [Gemorroj/Archive7z#11]
\Archive7z\Archive7z::getAutoCli on OSX returns cliWindows, because PHP_OS contains 'Darwin'
So, I think it will be good to fix it and add support for 7z on Mac (e.g. keka archiver https://github.com/aonez/Keka)

**Comment by Gemorroj** [comment]
https://github.com/Gemorroj/Archive7z/blob/4.0.0/src/Archive7z.php#L124
i can't reproduce problem.

```php
<?php
require __DIR__.'/vendor/autoload.php';

use Archive7z\Archive7z;

$obj = new Archive7z('vendor/gemorroj/archive7z/tests/fixtures/test.7z');

if (!$obj->isValid()) {
    throw new Exception('Incorrect archive');
}

var_dump($obj->getCli());
```

ouput:
```
string(39) "/usr/local/Cellar/p7zip/16.02_1/bin/7za"
```

**Comment by zakirovir6** [comment]
I'm sorry, it was in 2.0.0 release, not it works :)

**Comment by Gemorroj** [comment]
https://github.com/Gemorroj/Archive7z/compare/2.0.0...3.0.0#diff-9c2fac9bf2a0c278f25dfcf45de73c1cL124
yes, it fixed in `3.0.0` release
